src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,362 @@
+            nonActivePlayer.increaseStackPlyer(gameState.getPot());
+            nonActivePlayer.decreaseAmountInHand(nonActivePlayer.getAmountInHand());
+            activePlayer.decreaseAmountInHand(activePlayer.getAmountInHand());
+            gameState.decreasePot(gameState.pot);
+
+        }
+    }
+
+
+    public void updatePlayersOnShowdownResult(Player player1, Player player2, Boolean isDraw) {
+        player1.increaseStackPlyer(gameState.pot);
+        player1.decreaseAmountInHand(gameState.pot / 2);
+        player2.decreaseAmountInHand(gameState.pot / 2);
+        gameState.decreasePot(gameState.pot);
+        if (isDraw) {
+            player2.increaseStackPlyer(gameState.pot / 2);
+        }
+    }
+
+    public void seperateNextHandsAndShuffleCards() {
+        for (int i = 0; i < 10; i++) {
+            System.out.println();
+        }
+        try {
+            Thread.sleep(4000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        this.result = deck.shuffleDeckAndGetHandsAndBoard();
+        this.gameState.setGamePhase(GamePhase.PREFLOP);
+        player1.setHandPlayer(result.getHand1());
+        player2.setHandPlayer(result.getHand2());
+        firstDecisionSmallBlind = true;
+        playerFold = false;
+    }
+
+    public double amountToCall(Player player1, Player player2) {
+        return player1.getAmountInHand() - player2.getAmountInHand();
+    }
+
+    public void preFlopDetails(Player playerBigBlind, Player playerSmallBlind) {
+        System.out.println(result.getBoard());
+        System.out.println(playerBigBlind.getNamePlayer());
+        System.out.println(playerBigBlind.getHandPlayer());
+        System.out.println(playerBigBlind.getStackPlayer());
+        System.out.println(playerSmallBlind.getNamePlayer());
+        System.out.println(playerSmallBlind.getHandPlayer());
+        System.out.println(playerSmallBlind.getStackPlayer());
+        updatePotAndStackAmount(gameState.bigBlind, playerBigBlind);
+        updatePotAndStackAmount(gameState.smallBlind, playerSmallBlind);
+        playerSmallBlind.increasePlayerMoneyAtStrit(gameState.smallBlind);
+        playerBigBlind.increasePlayerMoneyAtStrit(gameState.bigBlind);
+        System.out.println("Zaczyna " + playerSmallBlind.getNamePlayer());
+        System.out.println(playerBigBlind.getNamePlayer() + " bigBlind " + gameState.bigBlind + "$");
+        System.out.println(playerSmallBlind.getNamePlayer() + " smallBlind " + gameState.smallBlind + "$");
+        printPot();
+        System.out.println(playerSmallBlind.getNamePlayer() + " podejmuje decyzje");
+        System.out.println(playerSmallBlind.getNamePlayer() + " " + amountToCall(playerBigBlind, playerSmallBlind) + "$ to call");
+        // System.out.println("1 = call 2 = raise");
+    }
+
+    public void printPot() {
+        System.out.println("Pot wynosi " + gameState.getPot() + "$");
+    }
```

Comment:
Taka dosc znaczaca zmiana kosmetyczna o ktorej nie gadalismy
ogolnie to ta klasa GamePlay jest dosc spora (co jest okej bo inaczej sie nie da)
ale mamy funkcje printPot(), printBoard(), amountToCall() ktore sa bardziej wspomagajace niz kluczowe.
Wiec zrob nowa klase GamePlayUtils i te 3 funckje ktore wypisalem wyciagnij je tam i zrob je zeby byly static